### PR TITLE
Fix to support AI-LIGHT with redis 6.0.6

### PR DESCRIPTION
**Pull request by alonre24**
Add `getRedisVersion` util, and use it instead of RM_getServerVersion which is not supported in old versions.

### CODE REVIEW

File: src/execution/DAG/dag.c [hunk 1]
```diff
@@ -685,9 +686,11 @@ void DAG_ReplyAndUnblock(RedisAI_OnFinishCtx *ctx, void *private_data) {
 
     RedisAI_RunInfo *rinfo = (RedisAI_RunInfo *)ctx;
     if (rinfo->client) {
-        if (RedisModule_GetServerVersion() >=
-            0x060200) { // The following command is supported only from redis 6.2
-            RedisModule_BlockedClientMeasureTimeEnd(rinfo->client);
+        int redis_version[3];
+        getRedisVersion(redis_version);
+        if (redis_version[0] >= 6 &&
+            redis_version[1] >= 2) { // The following command is supported only from redis 6.2
```
Comment (GuyAv46) [inline src/execution/DAG/dag.c:692]:
```suggestion
        if (redis_version[0] > 6 || // The following command is supported only from redis 6.2
           (redis_version[0] == 6 && redis_version[1] >= 2)) {
```

File: src/execution/command_parser.c [hunk 2]
```diff
@@ -71,8 +72,10 @@ int RedisAI_ExecuteCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int ar
         RedisModule_UnblockClient(rinfo->client, rinfo);
         return REDISMODULE_ERR;
     }
-    if (RedisModule_GetServerVersion() >=
-        0x060200) { // The following command is supported only from redis 6.2
+    int redis_version[3];
+    getRedisVersion(redis_version);
+    if (redis_version[0] >= 6 &&
+        redis_version[1] >= 2) { // The following command is supported only from redis 6.2
```
Comment (GuyAv46) [inline src/execution/command_parser.c:78]:
```suggestion
    if (redis_version[0] > 6 || // The following command is supported only from redis 6.2
       (redis_version[0] == 6 && redis_version[1] >= 2) {
```

**Review by GuyAv46 — CHANGES_REQUESTED** [review]

File: src/execution/utils.h [hunk 3]
```diff
@@ -11,7 +11,13 @@ bool VerifyKeyInThisShard(RedisModuleCtx *ctx, RedisModuleString *key_str);
...
  * Use this function when loading the model. Stores the version in global variables.
  */
-void getRedisVersion();
+void setRedisVersion();
+
+/**
+ * Returns redis version in the pre-allocated version_holder array as following: (major, minor,
+ * patch).
+ */
+void getRedisVersion(int *version_holder);
```
Comment (DvirDukhan) [inline src/execution/utils.h:20]:
```suggestion
void getRedisVersion(int *major, int *minor, int *patch);
```

File: src/execution/utils.h [hunk 4]
```diff
@@ -11,7 +11,13 @@ bool VerifyKeyInThisShard(RedisModuleCtx *ctx, RedisModuleString *key_str);
 /**
  * Use this function when loading the model. Stores the version in global variables.
  */
-void getRedisVersion();
+void setRedisVersion();
```
Comment (DvirDukhan) [inline src/execution/utils.h:14]:
```suggestion
void redisai_getRedisVersion();
```
if you change the name, let the reader know it is a module-level function

**Review by DvirDukhan — CHANGES_REQUESTED** [review]

File: src/execution/DAG/dag.c (hunk 1 above)
Comment (alonre24) [reply src/execution/DAG/dag.c:692]: Great catch, thanks!

**Review by GuyAv46 — COMMENTED** [review]
LGTM

**Review by DvirDukhan — APPROVED** [review]

**Review by GuyAv46 — APPROVED** [review]

**Pull request merged**